index.html

```diff
@@ -10,8 +10,119 @@
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorem qui quaerat sunt repellendus odio harum, aliquam accusantium atque! Placeat possimus repellat excepturi porro atque dicta quam, vel dolor repellendus consectetur.</p>
+                </div>
+                <div class="col-mobile-100">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorem qui quaerat sunt repellendus odio harum, aliquam accusantium atque! Placeat possimus repellat excepturi porro atque dicta quam, vel dolor repellendus consectetur.</p>
+                </div>
+            </div>
+            <div class="image_large image_right col-mobile-100 col-large-tablet-70">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+        </div>
+        <div class="three-columns row">
+            <div class="col-mobile-100 col-large-tablet-33">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+            <div class="col-mobile-100 col-large-tablet-33">
+                <div class="col-mobile-100">
+                    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ut totam quasi accusantium veritatis adipisci ipsa, sapiente enim fuga voluptate temporibus dicta dolores facere mollitia dolorem est ratione, doloribus distinctio hic.</p>
+                </div>
+                <div class="col-mobile-100">
+                    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ut totam quasi accusantium veritatis adipisci ipsa, sapiente enim fuga voluptate temporibus dicta dolores facere mollitia dolorem est ratione, doloribus distinctio hic.</p>
+                </div>
+            </div>
+            <div class="col-mobile-100 col-large-tablet-33">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+        </div>
+        <!-- TODO: implement slick slider -->
+        <div class="slider row">
+            <div class="col-mobile-100">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+        </div>
+        <!-- TODO: fix layout -->
+        <div class="three-columns shop row">
+            <div class="col-mobile-100 col-large-tablet-33">
+                <div class="col-large-tablet-50">
```

Comment:
w calym kontenerze musisz rowniez dodac klase col-mobile-100, inaczej mozesz miec bledy dla mobile, powtorz to we wszystkich kolumnach tego kontenera